Pollo/Models/ListDiffableModels/Poll.swift

```diff
@@ -147,9 +147,9 @@ class Poll: Codable {
         return letter == mcChoice
     }
 
-    func updateSelected(mcChoice: String) {
+    func updateSelected(mcChoice: String, choice: String) {
         guard let googleID = User.currentUser?.id else { return }
-        userAnswers[googleID]?[0] = PollChoice(text: mcChoice)
+        userAnswers[googleID]?[0] = PollChoice(letter: mcChoice, text: choice)
```

Comment:
ugh the whole letter vs. text thing is so annoying. hopefully soon backend will update and we'll only use indices for multiple choice options. but in the meantime this is a fine fix